components/PaywallModal.vue

```diff
@@ -0,0 +1,321 @@
+<template>
+  <UModal
+    :full-screen="props.isFullScreen || isMobileScreen"
+    :dismissible="props.isDismissible"
+    :ui="{ content: 'phone:rounded-2xl overflow-x-hidden phone:w-[90vw] max-w-[840px]' }"
+  >
+    <template #content>
+      <div class="flex flex-col phone:flex-row w-full h-full">
```

Comment:
I think phone doesnt not have enough space
```suggestion
      <div class="flex flex-col laptop:flex-row w-full h-full">
```